stock_picking_dispatch/models/stock_move.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# © 2012-2014 Alexandre Fayolle, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from openerp import fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+    dispatch_id = fields.Many2one(
+        'picking.dispatch', 'Dispatch', index=True, copy=False,
+        help='who this move is dispatched to'
```

Comment:
The help tooltip seems wrong, `dispatch_id` is not a partner thus `who` comes from a bad copy / paste I guess.